webpack.config.js

```diff
@@ -23,51 +18,47 @@ module.exports = {
       {
         test: /\.css$/,
         use: [
-          MiniCssExtractPlugin.loader,
           'css-loader',
           'postcss-loader',
         ]
       },
       {
-        test: /\.ico$\.woff$|\.ttf$|\.wav$|\.mp3$/,
-        loader: 'file-loader'  // <-- retain original file name
+        test: /\.(ico|woff|ttf|wav|mp3)$/,
+        type: 'asset/resource',
+        generator: {
+          filename: '[name][ext]', // keep original filename
+        },
        },
       {
-        test: /\.svg$/,
-        loader: 'svg-url-loader'
```

Comment:
the HtmlBundlerPlugin supports inline `svg` files in HTML and in CSS  out-of-the-box, you don't need any svg loader anymore